src/platform_impl/macos/window.rs

```diff
@@ -1472,6 +1478,21 @@ impl WindowExtMacOS for UnownedWindow {
       is_document_edited == YES
     }
   }
+
+  #[inline]
+  fn set_allows_automatic_window_tabbing(&self, enabled: bool) {
+    unsafe {
+      NSWindow::setAllowsAutomaticWindowTabbing_(*self.ns_window, if enabled { YES } else { NO })
```

Comment:
```suggestion
      self.ns_window.setAllowsAutomaticWindowTabbing_(if enabled { YES } else { NO })
```

Reply:
This doesn't work, `setAllowsAutomaticWindowTabbing_` only works when called as a static method, not an instance method.